pipeline/03_graph_db.py

```
@@ -1,11 +1,21 @@
 from dotenv import load_dotenv
 from neo4j import GraphDatabase
 from openai import OpenAI
 
+# Windows 콘솔(cp949) 에서도 한글 / em-dash 등 유니코드 출력 안전하게.
+# parser 와 동일한 처리 — 이게 없으면 print 한 줄에서 UnicodeEncodeError 로
+# 빌드 전체가 즉시 죽는다.
+try:
+    sys.stdout.reconfigure(encoding="utf-8", errors="replace")
+    sys.stderr.reconfigure(encoding="utf-8", errors="replace")
+except Exception:
+    pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)